client/src/components/layouts/default.tsx

```diff
@@ -177,21 +177,21 @@ class DefaultLayout extends Component<DefaultLayoutProps> {
           <link
             as='font'
             crossOrigin='anonymous'
-            href={robotoRegularURL}
+            href={hackZeroSlashRegularURL}
             rel='preload'
             type='font/woff'
           />
           <link
             as='font'
             crossOrigin='anonymous'
-            href={robotoBoldURL}
+            href={hackZeroSlashBoldURL}
             rel='preload'
             type='font/woff'
           />
           <link
             as='font'
             crossOrigin='anonymous'
-            href={robotoItalicURL}
+            href={hackZeroSlashItalicURL}
             rel='preload'
             type='font/woff'
           />
```

Comment:
Why only the woff files and not the woff2 files as well?

Reply:
If we preload the woff2 files then the type on these needs to be changed to `font/woff2`.

Reply:
We'll switch this to woff2 in a separate PR.